task/buildah/0.2/buildah.yaml

```diff
@@ -524,6 +526,7 @@ spec:
         echo -n "${IMAGE}@"
         cat "$(workspaces.source.path)/image-digest"
       } > "$(results.IMAGE_REF.path)"
+      cp "$(results.IMAGE_REF.path)" /shared/image_ref
```

Comment:
Is the copy needed?

Reply:
In testing, the `results.IMAGE_REF.path` wasn't available at the later task step so I copied the reference to the shared workspace so I could use it later.

I cannot find specific documentation for this behavior in Tekton.

Reply:
Hmm, I don't think Tekton behaves like that. Could you test it again? Maybe the nonexistent filepath was just caused by incorrect attempts at shell quoting

Reply:
Yes, certainly!

Reply:
I have removed this.